### PR TITLE
Bundle validate

### LINKED ISSUE #37: Can't create bundle
@AKamyshnikova I tried your new command added in your [last PR](https://github.com/cscetbon/casskop/pull/33) and it doesn't work atm

```
$ docker run --rm -ti -v $PWD:/go/casskop ghcr.io/cscetbon/casskop-build:latest bash 
root@61c08df6ae21:/go/casskop# kustomize build config/manifests | operator-sdk generate bundle -q --overwrite --version 2.1.5
/usr/local/bin/operator-sdk: line 1: Not: command not found
Error: unable to find one of 'kustomization.yaml', 'kustomization.yml' or 'Kustomization' in directory '/go/casskop/config/manifests'
```

it's the command `kustomize build config/manifests` that is failing

### PR DESCRIPTION
**Pull request by cscetbon**
| Q               | A
| --------------- | ---
| Bug fix?        | [X]
| License         | Apache 2.0

Fixes #37

### CODE REVIEW

**Comment by cscetbon** [comment]
@AKamyshnikova Can you confirm you don't see anything wrong here ?

**Comment by cscetbon** [comment]
Also running the same operations for multi-casskop do not work. Any idea why ?

File: Makefile [hunk 1]
```diff
@@ -170,10 +170,10 @@ endif
 bundle: generate
 	operator-sdk generate kustomize manifests -q;\
 	VERSION=$$(cat ./version/version.go | grep -Po '(?<=Version =\s").*(?=")');\
-	$(KUSTOMIZE) build config/manifests | operator-sdk generate bundle -q --overwrite --version $${VERSION} $(BUNDLE_METADATA_OPTS);\
-	operator-sdk bundle validate ./bundle;\
+	$(KUSTOMIZE) build config/crd | operator-sdk generate bundle -q --overwrite --version $${VERSION} $(BUNDLE_METADATA_OPTS)
```
Comment (AKamyshnikova) [inline Makefile:173]: I missed problem as kustomize was not configured properly, to be safe here - please add `make kustomize` before 173

Comment (cscetbon) [reply Makefile:173]: We don't need it anymore. I got rid of it and it comes from the ci image. We need to only use the docker image anytime we need to run something for safety

**Comment by cscetbon** [comment]
@AKamyshnikova what is https://github.com/cscetbon/casskop/pull/39/files#diff-fb9f73cd2716f1697ea42e5eab7ce7be376195b694a7553f774f7ee0d5555a5dR48-R50 used for ? I see it in our Makefile but I think it's not used anywhere. Can we just get rid of it ?

**Pull request merged**